benchmark/compare.js

```diff
@@ -70,8 +76,11 @@ if (showProgress) {
 (function recursive(i) {
   const job = queue[i];
 
-  const child = fork(path.resolve(__dirname, job.filename), cli.optional.set, {
-    execPath: cli.optional[job.binary],
+  const spawnArgs = ['-c', cpuCore, cli.optional[job.binary], path.resolve(__dirname, job.filename), ...cli.optional.set];
+  
+  const child = spawn('taskset', spawnArgs, {
+    env: process.env,
+    stdio: ['inherit', 'pipe', 'inherit', 'ipc'],
```

Comment:
the previous method should be used in case `-c` is not set.

Reply:
Done